### DISCUSSION TITLE
How to launch Web Widget through the Button

### DISCUSSION THREAD

**Discussion by Stephen-Lobo** [chatwoot/chatwoot#2772] [Q&A] [answered]
How to launch Web Widget Chat on the website through bootstrap button and jQuery ?

The script provided in the ChatWoot Inbox Setting triggers the Chat Window on onLoad Method, but instead of onLoad, we want to have customize button to trigger the Chat. Something like this :

`<a href="javascript:$zopim.chatwoot.window.show();">
   <img src="yourbutton.png" alt="Chat Button" />
</a>`

**Comment by pranavrajs** [comment, marked as answer]
https://www.chatwoot.com/docs/product/channels/live-chat/sdk/setup#trigger-widget-without-displaying-bubble

Please see this guide.

**Comment by Rishabhh17** [comment]
Hi @pranavrajs @Stephen-Lobo . I am doing the same and struggling a bit. I am having a button on my page and when a user click on that button only then I want to popup the widget without showing the bubble. I followed the documentation,  but didn't get success. Can anyone help me in this?
